.github/workflows/ci.yml

```diff
@@ -22,15 +22,11 @@ jobs:
       EASYPOST_PROD_API_KEY: ${{ secrets.EASYPOST_PROD_API_KEY }}
     strategy:
       matrix:
-        platform: [ 'NetStandard20', 'NetCore31', 'Net50', 'Net60' ]
+        platform: [ 'netstandard2.0', 'netcoreapp3.1', 'net5.0', 'net6.0' ]
```

Comment:
Do we need to change the casing here? Would prefer not to bug systems if we don't need to.

Reply:
Right now, I'm not seeing a way. By default, `msbuild` builds a structure like this:
<img width="343" alt="image" src="https://user-images.githubusercontent.com/17054780/163633616-9db8a986-0e1a-48be-8ee2-7e579694c266.png">

And I can't find a way to override the path. If I indicate a new folder path, it only makes a single DLL file in the indicated path. That won't work for us, because we explicitly want to test each platform version.

Reply:
Alright, I found a workaround to keep us from having to change the names of the required tests